vagrant-pxe-harvester/ansible/roles/harvester/templates/config-join.yaml.j2

```diff
@@ -1,5 +1,6 @@
 # example from https://github.com/harvester/ipxe-examples/blob/main/general/config-join.yaml
 
+scheme_version: 1
```

Comment:
We need to add these to `general/` examples as well.

Reply:
done